build/config/ios/ios_sim.py

```diff
@@ -100,8 +127,15 @@ def Main():
                       default='localhost:8080',
                       help='Sky server address.')
 
+  parser.add_argument('--ios_sim_path', dest='ios_sim_path', required=False,
```

Comment:
required is already false for all arguments starting with --.  You probably want this to be action='store_true'